source/jsonnet/northern-ireland/household/blocks/relationships/relationships_collector.jsonnet

```diff
@@ -103,16 +103,16 @@ local firstPersonNamePossessivePlaceholder = {
                 value: 'Husband or wife',
               },
               {
-                label: 'Same-sex civil partner',
+                label: 'Civil partner',
                 playback: {
-                  text: '{second_person_name} is your <em>Same-sex civil partner</em>',
+                  text: '{second_person_name} is your <em>civil partner</em>',
```

Comment:
Should these playbacks be capitalised?

Reply:
Just checked with donna and she said no as it's not a proper noun